.circleci/config.yml

```diff
@@ -2,6 +2,24 @@
 version: 2.1
 
 jobs:
+  run-setup:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - run:
+          name: Run scheduled workflow
+          command: |
+            curl --location --request POST 'https://circleci.com/api/v2/project/gh/aws-amplify/amplify-cli/pipeline' \
+            --header 'Content-Type: application/json' \
+            -u "${CCI_CONTINUATION_TOKEN}:" \
+            --data-raw '{
+              "branch": "master",
+                "parameters": {
+                  "setup": true,
+                  "nightly_console_integration_tests": false,
+                  "e2e_resource_cleanup": false
```

Comment:
Why is resource cleanup false?

Reply:
Because that job is triggered at a later time. These boolean parameters that are used for [here](https://github.com/aws-amplify/amplify-cli/blob/master/.circleci/config.yml#L10).